Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -302,6 +302,28 @@ static void bindSndio(Vector<CString>& args)
     bindIfExists(args, sndioHomeDir.get(), BindFlags::ReadWrite);
 }
 
+static void bindPipewire(Vector<CString>& args)
+{
+    const char* xdgRuntimeDir = g_get_user_runtime_dir();
+    const char* pwRuntimeDir = g_getenv("PIPEWIRE_RUNTIME_DIR");
+    const char* pwRemote = g_getenv("PIPEWIRE_REMOTE");
+
+    if (!pwRemote)
+        pwRemote = "pipewire-0";
+
+    if (pwRuntimeDir) {
+        GUniquePtr<char> pwRuntimeFile(g_build_filename(pwRuntimeDir, pwRemote, nullptr));
+        bindIfExists(args, pwRuntimeFile.get(), BindFlags::ReadWrite);
+    } else {
+        GUniquePtr<char> xdgRuntimeFile(g_build_filename(xdgRuntimeDir, pwRemote, nullptr));
+        bindIfExists(args, xdgRuntimeFile.get(), BindFlags::ReadWrite);
```

Comment:
Does it work if you bind it read-only? Probably? Confusingly, write permission is needed to delete or rename the socket, not actually required to write to it.